tests/PHPStan/Rules/Comparison/ElseIfConstantConditionRuleTest.php

```diff
@@ -134,10 +134,6 @@ public function testBug11674(): void
 				'Elseif condition is always false.',
 				28,
 			],
-			[
-				'Elseif condition is always false.',
-				36,
```

Comment:
Took me a while but I found the bad boy for which the elseif can be false: https://3v4l.org/p1sON